runtime/tests/org.eclipse.core.tests.harness/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Eclipse Core Tests Harness
 Bundle-SymbolicName: org.eclipse.core.tests.harness;singleton:=true
-Bundle-Version: 3.12.300.qualifier
+Bundle-Version: 3.13.0.qualifier
```

Comment:
Actually this is a good idea. But do PDE-API tools check versions?
Furthermore this project as API tools not enabled and as Aleks stated in https://github.com/eclipse-platform/eclipse.platform/pull/54#issuecomment-1188228511 these plug-ins are mainly for internal purposes and therefore don't have to follow the usual API rules. Therefore I think using tools will be inconvenient and without tooling support manual package-versioning is even more easy to forget mess up like bundle versioning (like I did).
So I think that is not suitable here.

Reply:
I don't mind about what API tools can/do but more whats recommended best practice for OSGi, if API tools miss support it should be added or replaced by a tool that can do better.

Reply:
Sure. I just think there are better places to start.
But if you insist. What do you think about the applied versions?
Furthermore it makes no sense to export the `org.eclipse.core.tests.session.samples` package.

Reply:
> if API tools miss support it should be added or replaced by a tool that can do better.

That would be very welcome. But in the meantime, we need to keep API Tools happy because it's what is serving us best.
As @HannesWell mentioned referencing @akurtakov statement, we don't enforce API correctness for tests at the moment, there is no requirement. But if someone is willing to start playing with package versioning in tests as a "sandbox" to progressively improves the overall practices of Platform, that would be welcome. But it's more a separate story/goal/issue and not a reason to delay other unrelated changes.

Reply:
>  There is always a beter place, a better time and a more demanding requirement to not start right now ;-)

On the other end, asking for more of Y on issues that are about X makes it longer for contributors to complete X and leave them less time to focus on Y. And additionally to this "quantative" impact, the impact on moral and motivation may create stress and exhaustion, which in the end reduce the efficiency of contributors.
I would really recommend to keep focus on the actual issues when doing reviews, and to move other discussions to other tickets.